gnuradio-runtime/lib/CMakeLists.txt

```diff
@@ -131,23 +128,22 @@ target_link_libraries(gnuradio-runtime PUBLIC
   Boost::thread
   Log4Cpp::log4cpp
   MPLib::mplib
-  ${libunwind_LIBRARIES}
+  pybind11::pybind11
```

Comment:
Or is pybind11 required because of this (new) statement?

Reply:
This `pybind11::pybind11` target only adds the include path for the pybind11 headers (and the Python headers, but we already do that anyway). So effectively it replaces the now-removed `target_include_directories` statement, and it maintains the functionality of communicating to downstream consumers of the `gnuradio-runtime` CMake target where they can find the pybind11 headers needed for `block_gateway.h`.